app/assets/javascripts/tylium/modules/local_auto_save.js

```diff
@@ -12,27 +11,39 @@ document.addEventListener("turbolinks:load", function() {
 
     if (data !== null) {
       for (let [key, value] of Object.entries(data)) {
-        form.querySelector(`[name='${key}']`).value = value
+        if (key.slice(-2) == "[]") {
+          value.forEach(function(checkboxValue) {
+            var $input = $(form).find(`[name="${key}"][value="${checkboxValue}"]`)
+            $input.prop("checked", true)
+          })
+        } else {
+          var $input = $(form).find(`[name="${key}"]`)
+          $input.val(value)
+          $input.trigger("load-preview")
+        }
       }
     } else {
       console.log("No data in localStorage for " + key);
     }
 
     // List of available inputs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input
     // Exclude these inputs so that it does not store unnecessary data in local cache
-    var excludedInputTypes = ["button", "file", "hidden", "image", "password", "reset", "submit"]
+    var excludedInputTypes = ["button", "file", "image", "password", "reset", "submit"]
```

Comment:
- Why are we saving hidden inputs?
- I believe we should use whitelisting here vs blacklisting

Reply:
@aapomm The hidden inputs are needed for the hidden tag input in issues#form. Currently the hidden input is changed using js when a dropdown option is selected.
![image](https://user-images.githubusercontent.com/18374402/80453692-5d62b280-895b-11ea-8de6-3808ed02d696.png)

Thanks, I've updated it to a whitelist instead.